stored_requests/fetcher.go

```diff
@@ -28,7 +29,7 @@ type Fetcher interface {
 
 type AccountFetcher interface {
 	// FetchAccount fetches the host account configuration for a publisher
-	FetchAccount(ctx context.Context, accountID string) (json.RawMessage, []error)
+	FetchAccount(ctx context.Context, accountDefaultJSON json.RawMessage, accountID string) (*config.Account, []error)
```

Comment:
I think we should leave the return type for `FetchAccount` to be a raw JSON string as our cache is currently configured to store stuff as JSON strings rather than objects. This is something we want ideally but IMHO it doesn't make sense until we change the cache code to accept objects as well.